Sources/Sentry/SentrySpanContext.m

```diff
@@ -27,8 +27,40 @@ - (instancetype)initWithtraceId:(SentryId *)traceId
         self.spanId = spanId;
         self.parentSpanId = parentId;
         self.sampled = sampled;
+        self.operation = @"";
+        self.status = kSentrySpanStatusUndefined;
```

Comment:
m: why do we need to set those default values if these are nullable fields?

Reply:
this is the behavior in .NET

Reply:
Well, not the status, but I prefer to avoid using pointer in enums

Reply:
I dont see it here https://github.com/getsentry/sentry-dotnet/blob/3bc6c66a14e653982a6a25df033d68d8ad12c4af/src/Sentry/Protocol/SpanStatus.cs

if we keep the "" as it is, we'd need to check that before serializing the JSON

Reply:
Ive raised this in .NET btw https://github.com/getsentry/sentry-dotnet/issues/773

Reply:
> I dont see it here https://github.com/getsentry/sentry-dotnet/blob/3bc6c66a14e653982a6a25df033d68d8ad12c4af/src/Sentry/Protocol/SpanStatus.cs
> 
> if we keep the "" as it is, we'd need to check that before serializing the JSON

You are right. I was referring to the default operation value. I will make it nullable